.github/workflows/cron-stale-issue.yml

```diff
@@ -15,6 +15,9 @@ jobs:
     steps:
       - uses: actions/stale@v7
         with:
-          days-before-stale: 60
-          include-only-assigned: true
```

Comment:
Removed this because we currently have lots of PRs that are not assigned to anyone (to be fixed as workflow improvement later).